### PR TITLE
bundle: bump to 4.11

### PR DESCRIPTION
**Pull request by Tal-or**
- Makefile: bump VERSION to 4.11
- Run ```make bundle```

### CODE REVIEW

**Comment by Tal-or** [comment]
/hold
Need more changes under release repo

**Comment by Tal-or** [comment]
/retest
now that https://github.com/openshift/release/pull/26142 got merged

**Comment by Tal-or** [comment]
@fromanirh Please retest CI E2E/e2e, I don't have a permission to do so (which BTW I think I should)

**Comment by fromanirh** [comment]
> @fromanirh Please retest CI E2E/e2e, I don't have a permission to do so (which BTW I think I should)

I think so as well. Let's check how GH actions permissions work, but AFAIK the only option is making you co-owner of the repo, which I'm fine with.

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -13,4 +13,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: quay.io/openshift-kni/numaresources-operator
-  newTag: 4.10.999-snapshot
+  newTag: 4.11.999-snapshot
```
Comment (fromanirh) [inline config/manager/kustomization.yaml:16]: please check if 
```
config/manifests/bases/numaresources-operator.clusterserviceversion.yaml
```
needs updates as well

Comment (Tal-or) [reply config/manager/kustomization.yaml:16]: Done

**Review by fromanirh — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**